driver/src/main/java/eu/cloudnetservice/cloudnet/driver/permission/PermissionGroup.java

```diff
@@ -93,30 +130,65 @@ protected PermissionGroup(
       .groupPermissions(group.groupPermissions());
   }
 
+  /**
+   * Gets the prefix of the group.
+   *
+   * @return the prefix of the group.
+   */
   public @NonNull String prefix() {
     return this.prefix;
   }
 
+  /**
+   * Gets the color of the group.
+   *
+   * @return the color of the group.
+   */
   public @NonNull String color() {
     return this.color;
   }
 
+  /**
+   * Gets the suffix of the group.
+   *
+   * @return the suffix of the group.
+   */
   public @NonNull String suffix() {
     return this.suffix;
   }
 
+  /**
+   * Gets the display of the group.
+   *
+   * @return the display of the group.
+   */
   public @NonNull String display() {
     return this.display;
   }
 
+  /**
+   * Gets the prefix of the group.
+   *
+   * @return the prefix of the group.
+   */
   public int sortId() {
     return this.sortId;
   }
 
+  /**
+   * Gets whether this group is the default group or not.
```

Comment:
Maybe add a note that there might be multiple groups which are marked as default, but always the first group (non-deterministic) gets choosen when the default group is needed